Agent/HandledException/NRMAHexBackgroundUploader.mm

```diff
@@ -3,46 +3,46 @@
 // Copyright © 2023 New Relic. All rights reserved.
 //
 
-#import "NRMAHexBackgroundUploader.h"
-#import "NRMASessionIdentifierManager.h"
-
-@interface NRMAHexBackgroundUploader ()
-@property(strong) NRMASessionIdentifierManager* sessionIdManager;
-@property(strong) NSURLSession* session;
-@end
-
-@implementation NRMAHexBackgroundUploader
-
-- (instancetype) initWithHexHost:(NSString*)hexHost {
-    self = [super init];
-    if (self) {
-        self.sessionIdManager = [[[NRMASessionIdentifierManager alloc] init] autorelease];
-        NSURLSessionConfiguration* backgroundConfiguration = [NSURLSessionConfiguration backgroundSessionConfigurationWithIdentifier:[self.sessionIdManager sessionIdentifier]];
-        self.session = [NSURLSession sessionWithConfiguration:backgroundConfiguration
-                                                delegate:self
-                                           delegateQueue:[NSOperationQueue currentQueue]];
-        self.hexHost = hexHost;
-    }
-    return self;
-}
-
-- (void) sendData:(NSData*)data {
-
-    NSMutableURLRequest* request = [self newPostWithURI:self.hexHost];
-    request.HTTPMethod = @"POST";
-    request.HTTPBody = data;
-    [request setValue:@"application/octet-stream" forHTTPHeaderField:@"Content-Type"];
-    NSURLSessionUploadTask* uploadTask = [self.session uploadTaskWithStreamedRequest:request];
-    [uploadTask resume];
-}
-
-- (void) dealloc {
-    [self.session invalidateAndCancel];
-    [self.session release];
-    [self.hexHost release];
-    [self.sessionIdManager release];
-
-    [super dealloc];
-}
-
-@end
+//#import "NRMAHexBackgroundUploader.h"
```

Comment:
If the contents of this file are going away, it would be better to simply delete it.

Reply:
Deleted.